From eb54234256b3a123ff9f0d4dff37a59f8ec080f5 Mon Sep 17 00:00:00 2001
From: Michael Lass <bevan@bi-co.net>
Date: Tue, 31 Oct 2023 20:29:12 +0100
Subject: [PATCH] Update version to 0.9

Although it's too late to hit the 0.9 release, update TRURL_VERSION_TXT to 0.9 so that builds from git reflect the current version.
---
 version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.h b/version.h
index 79e2341f..50b75812 100644
--- a/version.h
+++ b/version.h
@@ -24,6 +24,6 @@
  *
  ***************************************************************************/
 
-#define TRURL_VERSION_TXT "0.8"
+#define TRURL_VERSION_TXT "0.9"
 
 #endif
